src/main/java/org/mskcc/oncokb/curation/security/CustomOAuthSuccessHandler.java

```diff
@@ -56,8 +57,10 @@ public void onAuthenticationSuccess(HttpServletRequest request, HttpServletRespo
         // When we use the OncoKB public user table, then an activated user is not neccesarily
         // allowed to access this service. We will need to add a ROLE_CURATOR role to public
         // and check that the role exists here.
-        if (optionalUser.isPresent() && optionalUser.get().isActivated() && !optionalUser.get().getAuthorities().isEmpty()) {
-            UserDTO user = optionalUser.get();
+        if (
+            optionalUser.isPresent() && optionalUser.orElseThrow().isActivated() && !optionalUser.orElseThrow().getAuthorities().isEmpty()
+        ) {
+            UserDTO user = optionalUser.orElseThrow();
```

Comment:
Move this line outside the if statement and then can re-use `user` in the condition.

Reply:
I prefer not. It will throw exception if the optional is empty. To deal with that, I have to create another if condition which is a big unnecessary.